core/domain/acl_decorators.py

```diff
@@ -68,6 +76,39 @@ def check_activity_accessible(
             activity_rights.viewable_if_private)
 
 
+def check_exploration_editable(user_id, user_actions, exploration_rights):
+    """Returns a boolean to signify whether given exploration is editable
+    by the user or not.
+
+    Args:
+        user_id: str. Id of the given user.
+        user_actions: list(str). List of actions given user can perform.
+        exploration_rights: Exploration rights object.
```

Comment:
In the callers of this function, sometimes you check whether this is None, and sometimes you don't. Can we be consistent?

Would it make sense to pass in the exp_id and do all the retrieval and checking of exploration_rights here?